### PR TITLE
Column group resizing

### PR DESCRIPTION
**Pull request by Cuiyansong**

### CODE REVIEW

File: addon/views/header-group.js [hunk 1]
```diff
@@ -5,9 +5,7 @@ export default HeaderBlock.extend({
 
   classNameBindings: [ 'columnGroup.groupStyle' ],
 
-  width: Ember.computed(function() {
-    return this.get('columnGroup').get('savedWidth');
-  }),
+  width: Ember.computed.alias('columnGroup.savedWidth'),
```
Comment (jefflage) [inline addon/views/header-group.js]: Is `savedWidth` a new property you added?  If so, how does it differ from `width` and why did you need to add it?

Comment (cyjia) [inline addon/views/header-group.js]: Both `savedWidth` and `width` are existing properties in ColumnDefinition, `width` is defined as an alias get of `savedWidth` and is not expected to be overridden. Both `width` and `savedWidth` will be updated when resizing a column. That is, on init, `width` is an alias get and has the same value as `savedWidth`,  whenever resizing happens, `width` is a value and is kept in sync with `savedWidth`.

Comment (cyjia) [inline addon/views/header-group.js]: In ColumnGroupDefinition, `savedWidth` is overridden to be derived from `savedWidth` of all inner columns, and `width` is an alias get of `savedWidth` as defined in ColumnDefinition. Whenever resizing happens an inner column's resize method will be called, and the resize method will change `width` and `savedWidth` of that column to a new value, then the `savedWidth` of ColumnGroupDefinition will be updated(it is a computed property depends on `savedWidth` of inner columns). The `width` of a ColumnGroupDefinition is always an alias get of `savedWidth`, it's never modified to a value as what happens to a ColumnDefinition.
